src/WWTMVC5/Web.config

```diff
@@ -71,7 +77,12 @@ Content-Type: application/x-wt-->
       <add name="gettourfile" verb="*" path="GetTourFile.aspx" type="WWTMVC5.WwtWebHttpHandler, WWTMVC5" preCondition="managedHandler"/>
       <add name="gettourfile2" verb="*" path="GetTourFile2.aspx" type="WWTMVC5.WwtWebHttpHandler, WWTMVC5" preCondition="managedHandler"/>
     </handlers>
-    <modules runAllManagedModulesForAllRequests="false"/>
+    <modules runAllManagedModulesForAllRequests="false">
+      <remove name="Session" />
+      <add name="Session"
+           type="Microsoft.AspNet.SessionState.SessionStateModuleAsync, Microsoft.AspNet.SessionState.SessionStateModule, Version=1.1.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35"
```

Comment:
Interesting - it's package version is 4.0.1, but the assembly version 1.1.0.0. That's always annoying when they don't align.

Reply:
done